packages/angular-workspace/.eslintrc.js

```diff
@@ -119,12 +119,24 @@ module.exports = {
                         // The following attributes should be marked i18n in production applications but
                         // are ignored here as they do not actually need to be marked for tests / example apps
```

Comment:
I've updated these docs, can you merge from main, review the docs, and see if we need follow-up her before moving to owners?